### ISSUE TITLE
What is a "multi-selectable tablist"?

### ISSUE THREAD

**Issue by JAWS-test** [w3c/aria#1355]
https://w3c.github.io/aria/#tablist:

> For a multi-selectable tablist, authors SHOULD ensure each visible tabpanel has its aria-expanded attribute set to true...

From the definition of a tablist I am familiar with, I cannot imagine that it allows multiple selection, because only one tabpanel can be visible at a time. Unfortunately, many frameworks confuse tablist with [disclosure](https://w3c.github.io/aria-practices/examples/disclosure/disclosure-faq.html) or [accordion](https://w3c.github.io/aria-practices/examples/accordion/accordion.html) (where multiple selection is possible). Using tablist with these two elements, however, leads to the fact that they are not operable correctly, because the tabpanel is nested in the tablist.

Related: https://github.com/w3c/aria/issues/1032

**Comment by jnurthen** [comment]
We should probably deprecate this. It was to support the original accordion pattern which was based on tablist. I propose we deprecate this in 1.3 and remove in 1.4

**Comment by jnurthen** [comment]
Alternative proposal: leave aria-multiselectable but make it clear that this is only to be used when an operation can be performed on multiple tabs. For example, a browser allows you to select multiple tabs and close them all. aria-multiselectable could allow such an interface to be created.

No matter the result we should remove the 3rd paragraph entirely as it doesn't not add anything useful

> For a single-selectable tablist, authors SHOULD hide other tabpanel elements from the user until the user selects the tab associated with that tabpanel. For a multi-selectable tablist, authors SHOULD ensure each visible tabpanel has its aria-expanded attribute set to true, and that the remaining hidden tabpanel elements have their aria-expanded attributes set to false.

If we decide to allow aria-multiselectable we should add text to clarify that this allows a user to select tabs to perform other operations, not to expand multiple tabs at the same time.

**Comment by JAWS-test** [comment]
> Alternative proposal: leave aria-multiselectable but make it clear that this is only to be used when an operation can be performed on multiple tabs.

The question then would be, how to distinguish between selecting a tab and displaying a tab? [Currently aria-selected is used for the displayed tab](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html). If your suggestion is accepted it would be better to

* use aria-selected only for the selection and
* use aria-expanded on tab to indicate which tab panel is currently displayed.

Thus, the 3rd paragraph that you propose to delete would have to be kept, but reworded

**Comment by chlane** [comment]
@JAWS-test I agree that selection and visibility needs to be conveyed to users and that aria-expanded is the best fit for conveying visibility. I think the main thing we need to clarify are the concepts of selection and expansion when using multi-selectable tabs.

**Comment by chlane** [comment]
@jnurthen @JAWS-test It looks like `aria-current` could be a fit here because `current` and `selected` can have different meanings. See https://www.w3.org/TR/wai-aria-1.1/#aria-current.

**Comment by JAWS-test** [comment]
Whether we decide to use aria-current or aria-expanded, the thing should be backward compatible, i.e. aria-selected should still work unless the other attribute is used
